src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -7,7 +7,51 @@ class HelloWorldAnonymousClasses {
         fun greetSomeone(someone: String): String
     }
 
+    companion object {
```

Comment:
Could be just constants, however if it is 'detect' plugin requirements let it be )